src/__tests__/fixtures/static/font-config.js

```diff
@@ -6,10 +6,24 @@
  * @flow
  */
 
-import type {FontType} from '../../../types.js';
+import type {
+  AtomicFontsObjectType,
+  StyledFontsObjectType,
+} from '../../../types.js';
 
-export const preloadDepth: number = 1;
-export const fonts: {[string]: FontType} = {
+export function getFontConfig(
+  withStyleOverloads: boolean,
+  preloadOverrides?: {}
+) {
```

Comment:
Is there a concrete type for this config object that folks can use?  Ideally they could write their own and Flow would complain at the source if they did something like:

```
import type {FontConfigType} from '...';

const config: FontConfigType = {
   ...
};
```